user_manual.md

```diff
@@ -535,104 +632,131 @@ For these operators, we provide the equivalent formulation.
 A signature defining these files can be found in [non-core-eval](#non-core-eval).
 Note however that the evaluation of these operators is handled by more efficient methods internally in the Ethos, that is, they are not treated as syntax sugar internally.
 
-Core operators:
+### Core operators
+
 - `(eo::is_eq t1 t2)`
-    - Returns `true` if `t1` is (syntactically) equal to `t2`, or `false` if `t1` and `t2` are distinct and ground. Otherwise, it does not evaluate.
+  - Returns `true` if `t1` is (syntactically) equal to `t2`, or `false` if `t1` and `t2` are distinct and ground. Otherwise, it does not evaluate.
+
+ <!--CT To be discussed with Andy
```

Comment:
This looks fine to me, we should add this formalism to the other operators as well.